### ISSUE TITLE
Depth map and ground-truth size in Kitti dataset

### ISSUE THREAD

**Issue by jorgeturriate** [Lavreniuk/SPIdepth#22]
Dear Lavreniuk,

Thanks again for your research and contribution with Monocular Depth Estimation. I am trying to perform some tests with my training on Kitti, so I want to change the backbone and modify some hyperparameters to check the convergence. One of these changes consists on resizing the images. I modified the width and height specified in ./arg_files/hisfog/kitti/cvnXt_H_320x1024.txt, but the problem that I noticed is that in the Dataset class (KittiDataset), the variable self.full_res_shape is hard coded as (1242, 375), instead of reading those values from the txt file.

![Image](https://github.com/user-attachments/assets/8fca55a7-4d94-477c-be0f-e67805bdaf85)

These values are used when we work with the depth maps.

![Image](https://github.com/user-attachments/assets/7d86bbb3-276c-46a0-94d0-cd7235734a1c)

![Image](https://github.com/user-attachments/assets/c7193aac-0a79-4e35-b52b-53c837da58b7)

So, my question is why did you decide to use that fixed res_shape for the depth maps?

**Comment by Lavreniuk** [comment]
Hi, @jorgeturriate, to be honest, we took all the data loaders as they were and did not change them.
The parameters like width and height in the `.txt` file are mainly used for resizing images during training, especially for loss calculation. Our guess is that `self.full_res_shape` is hard-coded to maintain compatibility with KITTI’s native camera calibration.

**Comment by jorgeturriate** [comment]
Thanks for your response. I noticed that you already considered that shape in your `compute_depth_losses` function in the `trainer.py` file

![Image](https://github.com/user-attachments/assets/20db68cd-8c3b-4c94-9846-c843b0fb9f88)

So, I think there will not be any problem in the loss calculation.

Best regards.
